lib/opensusebasetest.pm

```diff
@@ -312,6 +312,14 @@ sub investigate_yast2_failure {
         "<5>.*Path.*on medium"                  => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
         "<5>.*Aborting requested by user"       => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
         "<5>.*Exception.cc"                     => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
+        #regressions found https://openqa.suse.de/tests/3646274#step/logs_from_installation_system/412
```

Comment:
I guess better to use build number, build is not important and will be vanished relatively soon.

Reply:
done.